test/integration-setup.ts

```diff
@@ -16,10 +16,6 @@ export function testCacheName(): string {
   return name + v4();
 }
 
-function useCompression(): boolean {
-  return process.env.COMPRESSION === 'true';
-}
```

Comment:
I am worried that by getting rid of this you are removing the CI test coverage for the compression case.

I think you should keep this function, and keep the `SetupIntegrationTest` function as a no-arg function, and just add the value of `useCompression` to the return value of `SetupIntegrationTest`. Then in your actual increment tests, you can check the value of `useCompression`, and then only run the appropriate tests. Will have to dig through the jest docs and see if there is a way to `skip` based on a boolean, which would be ideal, but otherwise we might just have to do `if (!bool) return`.